hack/format-scripts.sh

```diff
@@ -20,11 +20,20 @@ set -euo pipefail
 
 gopath="$(go env GOPATH)"
 
+install_shfmt() {
```

Comment:
Idea for the future: we could extract this little helper. I think that same snippet appears in about 4 different locations by now.

Reply:
This one specifically is in 2 places but there are a few more. Yes, I agree.

Reply:
To track, created https://github.com/kubernetes-sigs/krew/issues/332